general protection fault in path_init
